CHANGELOG-3.6.md

```diff
@@ -0,0 +1,17 @@
+
+
+Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.5.md).
```

Comment:
lgtm to start with the new changelog. We should be adding recommendation on versions as usual but it's OK to do it later.

Reply:
Should I add 3.2.28, 3.3.18 and 3.4.2 like in the other CHANGELOGs?

Reply:
Thanks for the quick update @avorima. About recommended versions we should list `3.3.18+, 3.4.2+ and 3.5.0+` but please hold on it for now - cc @gyuho @hexfusion @ptabor I will have it discussed in monthly meeting tomorrow if needed to wait by then. Thanks!

Reply:
@avorima we had it discussed in the community meeting today and we are waiting on few feedback, so we will add versions detail separately.